lib/Storage/MongoDbTimeSeriesStore.php

```diff
@@ -63,6 +64,12 @@ class MongoDbTimeSeriesStore implements TimeSeriesStoreInterface
 
     private $periodTable = 'period';
 
+    private $stats = [];
```

Comment:
A quick comment here would be useful, so that we know what these variables are intended for.
Also (nitpick) but I don't think the naming is quite right - for example `$widgetIds` actually contains widget names, right?